octobot_trading/exchanges/connectors/ccxt/ccxt_connector.py

```diff
@@ -373,9 +373,46 @@ async def create_limit_sell_order(self, symbol, quantity, price=None, params=Non
         )
 
     async def create_market_stop_loss_order(self, symbol, quantity, price, side, current_price, params=None) -> dict:
+        if self.client.has.get("createStopMarketOrder"):
+            try:
+                return self.adapter.adapt_order(
+                    await self.client.createStopMarketOrder(
+                        symbol,
+                        side=side,
+                        amount=quantity,
+                        stopPrice=price,
+                        params=params,
+                    )
+                )
+            except ccxt.OrderImmediatelyFillable:
```

Comment:
Is this behavior documented somewhere in the ccxt documentation?

Reply:
Havent found it documented anywhere, but if you run something like arbitrage mode, this could get raised when you have volatility. You were actually ending up with no stop loss order getting placed or triggered